src/features/cellnav/js/cellnav.js

```diff
@@ -888,8 +888,12 @@
                   var values = [];
                   var currentSelection = grid.api.cellNav.getCurrentSelection();
                   for (var i = 0; i < currentSelection.length; i++) {
-                    var cellDisplayValue = getAppendedColumnHeaderText(currentSelection[i].col) + getCellDisplayValue(currentSelection[i]);
-                    values.push(cellDisplayValue)
+                    var cellValue = getCellDisplayValue(currentSelection[i]);
+                    if (cellValue === '') {
+                        cellValue =  i18nService.getSafeText('headerCell.aria.empty');
+                    }
+                    var cellDisplayValue = cellValue + getAppendedColumnHeaderText(currentSelection[i].col);
```

Comment:
Why not store currentSelection[i] since we are accessing it twice

Reply:
I guess I was mostly worried about the undefined index issue but I doubt that would happen here

Reply:
It would also be nice to get the text once instead of constantly

Reply:
(preferable in like the constructor, since this never changes)